vault.go

```diff
@@ -414,6 +414,12 @@ func (r VaultService) get(rn *watchedResource) error {
 				secret, err = r.client.Logical().Read(rn.resource.path)
 			}
 		}
+		// if there is a top-level metadata key this is from a v2 kv store
```

Comment:
adding a link to docs for reference https://www.vaultproject.io/api/secret/kv/kv-v2.html